src/const.tsx

```diff
@@ -0,0 +1,84 @@
+
+export type OffersProps = {
+  id: string;
+  title: string;
+  type: string;
+  price: number;
+  city: {
+    name: string;
+    location: {
+      latitude: number;
+      longitude: number;
+      zoom: number;
+    };
+  };
+  location: {
+    latitude: number;
+    longitude: number;
+    zoom: number;
+  };
+  isFavorite: boolean;
+  isPremium: boolean;
+  rating: number;
+  description: string;
+    bedrooms: number;
+    goods: [
+      string
+    ];
+    host: {
+      name: string;
+      avatarUrl: string;
+      isPro: false;
+    };
+    images: [
+      string
+    ];
+    maxAdults: number;
+}
```

Comment:
Тут можно порефакторить и сделать код более читаемым и удобным:

```suggestion
 export type Location = {
      latitude: number;
      longitude: number;
      zoom: number;
 }

 export type City = {
     name: string;
    location: Location;
 }

export type OffersProps = {
  id: string;
  title: string;
  type: string;
  price: number;
  city: City;
  location: Location;
  ...
}
```

Reply:
И ты кстати уверен, что тебе у оффера нужно иметь `zoom` в Location и latitude/longitude в City в нём же? Для чего это нужно?